src/primitives/transaction.h

```diff
@@ -11,6 +11,8 @@
 #include "serialize.h"
 #include "uint256.h"
 
+static const unsigned int THEUNFORKENING = 257000;
```

Comment:
Yeah, I initially had it in main.h, but found it was required in the interest calculation code here to fix the interest rate bug.